File: 85.js
function activitySelection(start, end){
    let activities = start.map((s,i)=>[s,end[i]]);
    activities.sort((a,b)=>a[1]-b[1]);
    let count = 0, last_end = -1;
    for(let [s,e] of activities){
        if(s >= last_end){
            count++;
            last_end = e;
        }
    }
    return count;
}

// Driver code
let start = [1,3,0,5,8,5];
let end = [2,4,6,7,9,9];
console.log("Maximum activities:", activitySelection(start, end));

File: 85.py
def activity_selection(start, end):
    activities = sorted(zip(start, end), key=lambda x: x[1])
    count = 0
    last_end = -1
    for s, e in activities:
        if s >= last_end:
            count += 1
            last_end = e
    return count

# Driver code
if __name__ == "__main__":
    start = [1, 3, 0, 5, 8, 5]
    end = [2, 4, 6, 7, 9, 9]
    print("Maximum activities:", activity_selection(start, end))
